composer.json

```diff
@@ -10,10 +10,14 @@
             "homepage": "https://www.jobcloud.ch/c/en/about/team/?team=13"
         }
     ],
+    "repositories": [
+        {"type": "vcs", "no-api": true, "url": "git@github.com:jobcloud/php-kafka-schema-registry-client.git"}
+    ],
     "require": {
         "flix-tech/confluent-schema-registry-api": "^6.0",
         "symfony/console": "^4.3",
         "pimple/pimple": "^3.2",
+        "jobcloud/php-kafka-schema-registry-client": "dev-feat/JMP-1517/kafka-registry-client",
```

Comment:
This is temporary, will be removed after review, or at the very end, and exchanged with tagged version